mmtrack/models/backbones/sot_resnet.py

```diff
@@ -164,6 +164,27 @@ class SOTResNet(ResNet):
     def __init__(self, depth, *args, **kwargs):
         assert depth == 50, 'Only support r50 backbone for sot.'
         super(SOTResNet, self).__init__(depth, *args, **kwargs)
+        # unfreeze the backbone parameters so that DDP can build all parameters
+        # into buckets.
+        self._unfreeze_stages()
```

Comment:
Do we need a bool variable (e.g. `unfreeze_backbone`) to control whether do `_unfreeze_stages`?